idaplugin/rematch/updater.py

```diff
@@ -18,6 +18,11 @@ def update():
   if not config['settings']['update']['autocheck']:
     return
 
+  if not config['login']['server']:
+    logger('updater').debug("skipping auto update because server is\
+                            not configured yet, is it your first time ?")
+    return
+
```

Comment:
I think it'll be better to have a "first_usage" field instead of assuming that out of existing fields, if it's something we want.